lib/mix/lib/mix/tasks/escript.install.ex

```diff
@@ -85,6 +85,11 @@ defmodule Mix.Tasks.Escript.Install do
       write_bat!(dst <> ".bat", :os.type)
 
       Mix.shell.info [:green, "* creating ", :reset, Path.relative_to_cwd(dst)]
+      case :os.type() do
```

Comment:
We should do this on the test environment maybe? We typically don't change the PATH but rather ask users to!